assets/sass/v2/_phone-challenge.scss

```diff
@@ -0,0 +1,20 @@
+.phone-authenticator-challenge {
+  .okta-form-subtitle {
+    text-align: center;
+  }
+  &__button--primary.link-button {
```

Comment:
link button already has class indicating whether they are primary or not
`button button-primary`
vs.
`button`

sure it may not follow BAM standard you mentioned before, but are able to leverage it for now?